src/components/Main/ModalWarning.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Button, Header, Icon, Modal, ModalProps } from 'semantic-ui-react'
 
 const ModalWarning: React.FC<ModalProps> = (showModal) => {
-  const { title, message, option, onClick, ...modalProps } = showModal
+  const { title, message, option, optionCancel, onClick, onCancel, ...modalProps } = showModal
```

Comment:
Minor addition to Modal options, surprised we didn't already have a "Cancel" button on these Modals. I think we should move to a global Modal UI that works the same was as the [ToastProvider](#1189), but that's for another day.

Reply:
The cancel used to be "click outside"

Reply:
Yeah, and it still is. I just thought it was potentially confusing to not have an explicit "Cancel" button too. It's pretty standard for a confirmation dialog.